starcheck/src/run_regress

```diff
@@ -5,13 +5,15 @@
 RunRegression()
 # Run regression tests.
 # Arg 1 is the AGASC version (e.g. 1p5)
-# Arg 2 is the fid characteristics name (e.g. fid_CHARACTERIS_FEB07)
+# Arg 2 is the AGASC file for h5
```

Comment:
Technically, we need changes to run the regression tests if we want to support the new API in the test version of starcheck.  We don't *really* need this agasc_file change to have a home in the tests if we don't mind using agasc1p7.h5 when called from starcheck on these test dirs (which should be fine).

So we could just cut the agasc 1p5 tests and remove both the previous agasc_dir option and the new agasc_file option so everything runs at default values.